docs/getting-started/preview.md

```diff
@@ -0,0 +1,79 @@
+# W4H-Preview
```

Comment:
Move this all thing in the main README.md Then explain at the end that the rest of documentation is work in progress.